src/moments.jl

```diff
@@ -242,59 +242,59 @@ function _moment2(v::RealArray, wv::AbstractWeights, m::Real; corrected=false)
     varcorrection(wv, corrected) * s
 end
 
-function _moment3(v::RealArray, m::Real)
+function _moment3(v::RealArray{T}, m::Real) where T
     n = length(v)
-    s = 0.0
+    s = zero(T)
     for i = 1:n
         @inbounds z = v[i] - m
         s += z * z * z
     end
     s / n
 end
 
-function _moment3(v::RealArray, wv::AbstractWeights, m::Real)
+function _moment3(v::RealArray{T}, wv::AbstractWeights, m::Real; corrected=false) where T
```

Comment:
The added keyword argument here seems not to be used?

Reply:
copy-paste error